docs/dev/privacy-model-of-secret-contracts.md

```diff
@@ -468,17 +468,17 @@ pub fn handle<S: Storage, A: Api, Q: Querier>(
 }
 ```
 
-By looking at state write operation, an attacker can guess which function was called based on the size of the key that was used to write to storage:
+By looking at a state write operation, an attacker can guess which function was called based on the size of the key used to write to storage:
 
 1. `send`
 2. `transfer`
 
 Again, some quick fixes for this issue might be:
 
-1. Renaming `transfer` to `tsfr`.
-2. Padding `send` to have the same length as `transfer`: `sendsend`.
+1. Renaming `transfer` to `tsfr`
+2. Padding `send` to have the same length as `transfer`: `sendsend`
 
-```rust
+```rustrust
```

Comment:
typo.

there's got to be some way that I can simply go in and fix these small errors.